packages/apollo-link/package.json

```diff
@@ -34,6 +34,8 @@
     "watch": "tsc -w -p . & rollup -c -w"
   },
   "dependencies": {
+    "apollo-utilities": "^1.2.1",
```

Comment:
Same for these dependencies

Reply:
Hmm, I see the argument for `devDependencies`, but I'm a little worried about duplicating `apollo-utilities` in production bundles if we pin the exact version.

Reply:
Well, ofcourse it can't break because of a failing build since it's bundled in but well yes mostly up to how you want to handle these. I personally like having control over dependencies but that's my two cents on it.

Reply:
I agree application developers should use tools like `yarn.lock` or `package-lock.json` to pin all their dependencies, but from the perspective of this library, any version of `apollo-utilities` that exports `getOperationName` will work, which includes at least two years worth of past versions, and probably all future versions too.

Reply:
In that case I'm okay with it, I'll resolve this conversation